Explainer_application/explainer_system.py

```diff
@@ -0,0 +1,54 @@
+import asyncio
+import logging
+import time
+import files_handler.file_handler as fh
+from Explainer_application import explain_pptx
+
+UPLOAD_FOLDER = "../files_handler/uploads"
+OUTPUT_FOLDER = "../files_handler/outputs"
+
+
+def process_file(file_name: str):
+    """
+    Process a file.
+    :param file_name: str, name of the file to process.
+    :return: None.
+    """
+    try:
+
+        complete_file_name = UPLOAD_FOLDER + "/" + file_name + ".pptx"
+        asyncio.run(explain_pptx(complete_file_name, OUTPUT_FOLDER, file_name))
+        print(f"File {file_name} processed successfully")
+    except Exception as e:
```

Comment:
Always throw the most specific exception -
https://stackoverflow.com/questions/2052390/manually-raising-throwing-an-exception-in-python
https://docs.python.org/3/library/exceptions.html